torch/utils/data/dataloader.py

```diff
@@ -236,15 +244,20 @@ class DataLoader(object):
             (default: 0)
         collate_fn (callable, optional)
         pin_memory (bool, optional)
+        drop_last (bool, optional): set to ``True`` to drop remaining samples
+            that is not enough of a batch size. If False and the size of dataset
+            is not divisible by batch size, then the last batch will be smaller.
+            (default: False)
```

Comment:
set to True to drop the last incomplete batch, if the dataset size is not divisible by the batch size.